global_routes.go

```diff
@@ -249,3 +251,58 @@ func wagCircuitBreakerRoutes(fields map[string]interface{}) []decode.AlertRoute
 		},
 	}
 }
+
+var allowedLifecycleEvents = []string{
+	"app_deploying",
+	"app_scaling",
+	"app_rollback",
+	"app_stopping",
+	//"app_freezing",
+	//"app_unfreezing",
+	"app_restarting",
+	"app_canary",
+	"app_canaryscaling",
+}
+
+// App Lifecycle routes
+func appLifecycleRoutes(fields map[string]interface{}) []decode.AlertRoute {
+	category, ok := fields["category"].(string)
+	if !ok {
+		return []decode.AlertRoute{}
+	}
+
+	title, ok := fields["title"].(string)
+	if !ok {
+		return []decode.AlertRoute{}
+	}
+
+	if category != "app_lifecycle" || !listContains(allowedLifecycleEvents, title) {
+		return []decode.AlertRoute{}
+	}
+
+	return []decode.AlertRoute{
+		decode.AlertRoute{
+			Series: "app_lifecycle",
+			Dimensions: []string{
+				"container_app",
+				"container_env",
+				"launched_scope",
```

Comment:
Just curious what is launched_scope?

Reply:
it is "production" or "development". The use case is to roll up all dev envs into just "development"